console/my-account/concepts.mdx

```diff
@@ -71,15 +71,15 @@ A **M**edia **A**ccess **C**ontrol Address is a unique ID assigned to network in
 
 ## Magic Link
 
-A magic link provides quick and secure access to your account without the hassle of remembering your [password](#login-information). When you choose to sign in with a magic link, you receive a unique link sent directly to your email inbox which you can use one time only to authenticate your login. Afterward, it automatically becomes invalid.
+A magic link provides quick and secure access to your account without the hassle of remembering your [password](#login-information). When you choose to sign in with a magic link, you receive a unique link sent directly to your email inbox which you can use one time only to authenticate your login. Afterwards, it automatically becomes invalid.
```

Comment:
Afterward is correct here since we use US english :)

Reply:
My bad